cmd/cmdutils.go

```diff
@@ -85,3 +85,17 @@ func validateName(name string) error {
 	return nil
 
 }
+
+// validateStoragePath will validate storagePath, if there is any existing storage with similar path, it will give an error
+func validateStoragePath(client *occlient.Client, storagePath, componentName, applicationName string) error {
```

Comment:
test here ^^ Would it be possible to use fakeClient?

Reply:
added